### PR TITLE
.

### PR DESCRIPTION
**Pull request by StanislavBrizgalov**

### CODE REVIEW

**Comment by MikitaTsiarentsyeu** [comment]
Are so lazy that you cannot write a proper description for your request? I'm truly sick of those dots.

File: Tasks/Brizgalov_Stanislav_Tasks/Homework/tasks3/tasks3.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+q = int(input("Введите количество символов в строке болише 15: "))
+if q <= 15:
+    print("вы ввели неверное количество символов")
+else:
+    def func_1(a):
+        return len(" ".join(a))
+
+
+    def func_2(line):
+        spaces = q - sum(len(line2) for line2 in line)
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Brizgalov_Stanislav_Tasks/Homework/tasks3/tasks3.py:10]: It's a messy code, I will not believe that you do it to save some memory on the function objects. Next time just describe your function without it being dependent on validation of an input, Also for the sake of all gods, just name your functions according to their role in a solution, not just func_1. Your score for this task is 8.5.

File: Tasks/Brizgalov_Stanislav_Tasks/Homework/tasks4.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            else:
+                line += f'{start}-{stop}, '
+            start = x
+            stop = x
+    if stop == start:
+        line += f'{stop}'
+    else:
+        line += f'{start}-{stop}'
+
+    print(line)
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Brizgalov_Stanislav_Tasks/Homework/tasks4.py:25]: good, your score is 9.0.

File: Tasks/Brizgalov_Stanislav_Tasks/Homework/tasks5.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+def f(a):
+    if not a:
+        return a
+    if isinstance(a[0], list):
+        return f(a[0]) + f(a[1:])
+    return a[:1] + f(a[1:])
+
+
+s = [1, 2, [2, 4, [[7, 8], 4, 6]]]
+print(sum(f(s)))
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Brizgalov_Stanislav_Tasks/Homework/tasks5.py:10]: nice one, it's also 9.0.

**Review by MikitaTsiarentsyeu — APPROVED** [review]

**Comment by StanislavBrizgalov** [comment]
Извиняюсь за точки, заливал через  pycharm, не думал что так получится,
когда увидел сам на себя разозлился, на счёт функций, плохо знаю
английский, исправлюсь.

сб, 1 мая 2021 г., 15:20 MikitaTsiarentsyeu ***@***.***>:

**Pull request merged**